sources/web/datalab/polymer/components/item-list/item-list.html

```diff
@@ -93,9 +93,14 @@
       .column {
         flex: 2;
       }
+      #header > .column {
+        color: var(--primary-fg-color);
+        font-size: 13px;
+      }
       .column + .column {
         flex: 1;
         max-width: 150px;
+        color: var(--neutral-fg-color);
```

Comment:
Making the first column be bolder than the others makes me think the data is sorted by the first column. I think I would be happier with all the data being the primary color.

Reply:
I'm not sure I see why the darker color implies sort order. It just makes this column more prominent, which is the original intent.
This is also inspired by the Drive UI, which I think makes sense.